src/driver/experimental_trial.py

```diff
@@ -295,6 +296,18 @@ def _compute_single_publisher_fractions_dataframe(
             column_name = f"relative_error_at_{int(r*100):03d}"
             results[column_name] = [relative_error]
 
+        for f in range(2, max_frequency):
```

Comment:
How about f = 1? It seems not to hurt by recording f=1.

Reply:
The case f=1 was already covered by the previous code, so there was no need to duplicate it.  Nonetheless, I have changed this to start from f=1.